assets/src/components/ladderPage.tsx

```diff
@@ -210,6 +242,14 @@ const LadderPageWithTabs = (): ReactElement<HTMLDivElement> => {
               tab={routeTab}
               selectTab={() => dispatch(selectRouteTab(routeTab.uuid))}
               closeTab={() => dispatch(closeRouteTab(routeTab.uuid))}
+              showSaveIcon={!isPreset(routeTab) || isEditedPreset(routeTab)}
+              saveTab={() => {
+                if (isEditedPreset(routeTab)) {
```

Comment:
**thought:** If you passed the entire `routeTab`, could you remove the condition here, call e.g. `savePreset` all of the time, and let either the action or maybe better yet the reducer figure out the "create new vs update existing" question here?

Reply:
This is a really good point - since I'm going to be moving around these callbacks and code associated with them as part of implementing the modals I mentioned in another comment, I'll leave a note on that ticket to put this logic in the reducer while I'm at it.